tests/chflags/01.t

```diff
@@ -8,8 +8,6 @@ dir=`dirname $0`
 . ${dir}/../misc.sh
 
 require chflags
-if requires_root
```

Comment:
Requires root: superuser flags.

Reply:
I have removed the "Remove requires_root" patch from the PR for now. I believe it makes sense to tackle that topic in a separate PR.